proto/cosmos/tx/v1beta1/service.proto

```diff
@@ -42,6 +43,17 @@ message GetTxsEventRequest {
   repeated string events = 1;
   // pagination defines an pagination for the request.
   cosmos.base.query.v1beta1.PageRequest pagination = 2;
+  OrderBy order_by = 3;
+}
+
+// OrderBy defines the sorting order
+enum OrderBy {
+  // ORDER_BY_UNSPECIFIED specifies an unknown sorting order
+  ORDER_BY_UNSPECIFIED = 0;
```

Comment:
In this case, we can maybe use ORDER_BY_ASC as 0, since it's the default value.

Reply:
I'm getting proto linter error `Enum zero value name \"ORDER_BY_ASC\" should be suffixed with \"_UNSPECIFIED\"`

Reply:
ah ok, let's keep it like this then